src/main/java/games/strategy/engine/ClientFileSystemHelper.java

```diff
@@ -66,8 +65,7 @@ private static String getGameRunnerFileLocation(final String runnerClassName) {
 
 
   private static String getTripleaJarWithEngineVersionStringPath() {
-    final Version version = ClientContext.engineVersion();
-    return "triplea_" + version.toStringFull('_') + ".jar!";
```

Comment:
@ssoloff this was the only usage of the Version#toStringFull overload with the optional char param.

Reply:
:+1: Will add that to the follow-up PR.